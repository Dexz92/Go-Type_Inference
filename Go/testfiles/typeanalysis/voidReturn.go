func main() {
  helper1();
  helper2();
  helper3();
  return;
}

func helper1()  {
  return 1;
}

func helper2()  {
  return 2.0;
}

func helper3()  {
  return;
}
