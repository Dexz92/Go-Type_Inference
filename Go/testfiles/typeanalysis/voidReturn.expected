Error at line 9: Trying to return int in void function
Error at line 13: Trying to return float in void function
